LICENSE.md

```diff
@@ -1,6 +1,6 @@
 # MIT License
 
-Copyright (c) 2021-2022 Franck Nijhof
+Copyright (c) 2021-2023 Franck Nijhof
```

Comment:
This PR has unrelated changes to the base image bump (like this one, but many more).